### ISSUE TITLE
Classes

### ISSUE THREAD

**Issue by tehbaskin** [tehbaskin/realmcraftproj#2]
Need to draft classes

**Comment by dochyel** [comment]
For base classes, so far i have come up with 4 core ideas.  You have Mercenery as a sword & board foot soldier martial combatant, Rogue as a sneak/trap/skill class, Wizard as a caster type, Healer as a buffer/hp restore, and Politico as a basic interpersonal manager.

The Mercenary class will be the primary martial class, able to use most weapons and armor.  They should be able to develop as a master of one type of weapon, such as a sword saint or archer subclass.

Rogues will be fast and nimble, with high move speed and finesse based damage.  They can use light weapons and ranged weapons that have high critical chances, and they will be able to advance in trapmaking or assassination themed abilities.

The Wizard will be a utility caster with spell components that must be gathered and consumed for spell effects, for everything from summoning npc creatures, temporary structures like bridges or walls, teleportation, and high damage output spells.  The should be able to focus on creation or combat magic.

Healers are just that.  They will assist groups with hp and ither resource management like food, fatigue, etc.  They will be more buff based than Wizards.  They will be able to focus on either offensive or defensive stat increase spells.

Politicos will be the general man-about-town.  A politico will get bonus advancement from questig in early levels, and grant bonuses to other players at higher levels.  They will be able to focus as a Merhant Craftsman or an Aristocratic Management track, because in order to succeed in either of those paths, you need to know people to succeed.

**Comment by tehbaskin** [comment]
So is the thought we have 5 core concepts, and from there we build archetypes or subclasses?

**Comment by dochyel** [comment]
That is where i was headed with it.  If each of the core classes has a different power stat, each of the subclasses can have a different sub stat.  Like mercenary uses a str like primary stat, archer would make dex a sub stat.  Wizatrd uses int, blaster adds charisma as force of personality is added to offensive magics.  A politico thrives on charisma, but merchants need to be smart and aristocrats need to be wise.

**Comment by tehbaskin** [comment]
Also spell components are a cool thought, but not for all spells. Component requiring spells need to be such, just because they should have a rarity equal to the component, and should warrant the opportunity cost of gathering, and carrying said component.

The wizard should also have some offensive or defensive spells that use a focus, much as a martial character performs melee combat with an instrument. Albeit these spells should not be as inherently potent as melee combat from an equally equipped martial character, to compensate for the wizards additional capability to use more utility magic.

**Comment by dochyel** [comment]
Wizard spell components would be relative to the task.  Some bark or tinder to catch an arcane spark to light a campfire that burns longer and hotter than it should for example.  Generally the only spells that use rare components are big effects, suh as using a few crafting compoents for a temporary version of the thig like a bridge.

**Comment by dochyel** [comment]
Spell foci would be something like a magic wand or staff, with more difficult to create or more expensive versions granting larger bonuses to combat spell damage or maybe duration or range.

**Comment by tehbaskin** [comment]
Rogues would be potent in melee as well but the difference should be the requirement of positioning and situational limits to combat effectiveness.

Instead of healer, I like just going cleric, some divine class. That ties a pantheon into the world.

And instead of politico, what about going with arbiter, or advocate... where the class is a manipulator type, with skills related to promoting communications, reputations, arbiting social rolls, and affecting the willpower of others.

**Comment by tehbaskin** [comment]
Right, spell foci is effectively the staff or wand or whatever, and spell foci spells scale off of it the way melee scales off a weapon.

**Comment by tehbaskin** [comment]
The divine caster would be the same way, but with a divine focus object or weapon.

**Comment by dochyel** [comment]
The names of the classes are not as important as the concept, and i think you have nailed it so far.  I called it politico because it is politics at the personal level.  Rogues would be able to do loads of situational damage, whether by catching their target off guard from stealth/flank/etc, or the assassins can gain bonuses by not having to move, like a bow or crossbow sniper studying their target.  By not moving or the subject not being aware, the damage potential on the next strike improves, reducing resource costs like arrow/bolt materials.

**Comment by dochyel** [comment]
Divine casters will need a weapon, so allowing them a focus like a holy symbol that attaches to a weapon or shield would prevent the need for constant inventory juggling, and the quality of the symbol (wooden, stoe, iron, silver, gold, combinations, etc) can have effects on the divine spells that use the focus.

**Comment by Phraughst** [comment]
I want to nail down a setting so we can get themes. Would the political classes dominate leadership? Should they? I think they should. Their blatant weakness is bloody coup. I like the idea of components, degrading equipment, and ammo. All better for economy and creates nice space for a merchant/builder/producer class.

**Comment by dochyel** [comment]
The best part is that all of the classes can do anything, but they have different advantages.  A caster with unseen servant and floating disk can make up for slow harvest speeds based on physical stats by spending some spell components and spell points/prepared spell slots.  A cleric can feed a party with a single person's meal, like the loaves and fishes, but those things rot faster than non-magic versions.  Rogues can hunt and forage faster than melee combatants, while the merceneries will practically never tire.  Socialites are just plain better salesmen and can make up for lack of resource gatherig skill with resale value and mercantile skill.
